library/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/processor/WireframeUtilsTest.kt

```diff
@@ -746,11 +813,27 @@ internal class WireframeUtilsTest {
         )
     }
 
+    private fun Forge.aValidWireframe(): MobileSegment.Wireframe {
+        return when (val wireframe = getForgery<MobileSegment.Wireframe>()) {
+            is MobileSegment.Wireframe.ShapeWireframe ->
+                wireframe.copy(shapeStyle = forgeNonTransparentShapeStyle())
+            else -> wireframe
+        }
+    }
```

Comment:
seems like this method is not used (one from the companion object is used instead)